### ISSUE TITLE
Attributes don't work properly

### ISSUE THREAD

**Issue by LuckyDmitry** [krzysztofzablocki/Sourcery#1043]
Hi, I'm trying to use attributes to generate FeatureFlags for app. I use them as description. 
I retrive it like that.

`
<%_ for variable in type.variables { -%>
                <%_ if let newVal = variable.attributes["FeatureFlag"] { -%>
                    <%= "\(newVal[0])" %>
                <%_}%>
            <%_}%>
`
It works fine, I got my struct, but to extract remoteFeatureName for example I need to use **.arguments** like that 

**Struct before**

`
  @FeatureFlag(
            defaultValue: false,
            remoteFeatureName: "provision_enabled",
            path: "provisioning_features.is_enabled",
            description: nil)
        }
`

After  newVal[0].**.arguments**

`
["defaultValue": false, "remoteFeatureName": provision_enabled, "description": nil, "path": provisioning_features.is_enabled]
`

It removes quotes from strings and to fix it I can wrap this string, but I have more complex structres in which I can't do it.
Please, help me. How can I overcome it and extract these fields without losing the quotes?

**Comment by artem-alexandrov** [comment]
Wanted to use the same approach, but struggling in making `|replace: ",.*","",regex` to work. It seems like it doesn't use the mode modifier. It would be great if I could get arguments array, but I can't as well...

**Comment by krzysztofzablocki** [comment]
@artem-alexandrov might be old Stencil / StencilSwiftKit version, are you on newest Sourcery?

**Comment by artem-alexandrov** [comment]
Yep, it's 1.8.2.

Maybe it's something in the syntax, I'm trying to use it like this:

```
{% for variable in type["FeatureFlags"].variables where variable|!annotated:"skip" %}
    {% for attribute in variable.attributes %}
        {% set attributeString %}{{ variable.attributes[attribute] }}{% endset %}
        {{ attributeString|replace:",.*","",regex }}
    {% endfor %}
{% endfor %}
```

just using `attribute` doesn't work, it only has the attribute name, also `variable.attributes[attribute]|replace...` doesn't work with invalidInputType error too.

**Comment by art-divin** [comment]
Hi 👋🏻 

Found the logic at fault, when Attribute is parsed from `AttributeSyntax`, for some reason `"` is replaced with an empty string - unsure why this was applied during initial implementation, when `swift-syntax` was integrated by @krzysztofzablocki . As far as I can see, the behaviour changes a little, **but** it changes in a correct manner, that is, the reported issue is fixed.

I suspect there was an assumption made initially that quotes did not look good, especially when writing unit tests; however, as practice shows, they are needed.

```swift
extension Attribute {
    convenience init(_ attribute: AttributeSyntax) {
             ...
              let components = part.split(separator: ":", maxSplits: 1)
              switch components.count {
              case 2:
                  arguments[components[0].trimmed] = components[1].replacingOccurrences(of: "\"", with: "").trimmed as NSString
              case 1:
                  arguments["\(idx)"] = components[0].replacingOccurrences(of: "\"", with: "").trimmed as NSString
              ...
    }
   ...
}
```

I have removed this workaround, so, I suppose, this is fixed by #1316
